admin/google_search_console/views/gsc-display.php

```diff
@@ -8,6 +8,56 @@
 // Admin header.
 Yoast_Form::get_instance()->admin_header( false, 'wpseo-gsc', false, 'yoast_wpseo_gsc_options' );
 
+// GSC Error notification.
+$gsc_profile = WPSEO_GSC_Settings::get_profile();
+$gsc_url     = 'https://search.google.com/search-console/index';
+if ( $gsc_profile !== '' ) {
+	$gsc_url = 'https://search.google.com/search-console/index?resource_id=' . rawurlencode( $gsc_profile );;
+}
+$gsc_post_url            = 'https://yoa.st/google-search-console-deprecated';
+$gsc_style_alert         = '
+	display: flex;
+	align-items: baseline;
+	position: relative;
+	padding: 16px;
+	border: 1px solid rgba(0, 0, 0, 0.2);
+	font-family: \'Segoe UI\', Roboto, Oxygen-Sans, Ubuntu, Cantarell, \'Helvetica Neue\', sans-serif;
+	font-size: 14px;
+	font-weight: 400;
+	line-height: 1.5;
+	margin: 16px 0;
+	color: #450c11;
+	background: #f8d7da;
+';
+$gsc_style_alert_icon    = 'display: block; margin-right: 8px;';
+$gsc_style_alert_content = 'max-width: 600px;';
+$gsc_style_alert_link    = 'color: #004973;';
+$gsc_notification        = sprintf(
+/* Translators: %1$s: expands to opening anchor tag, %2$s expands to closing anchor tag. */
+	__( 'Google has discontinued its Crawl Errors API. Therefore, any possible crawl errors you might have cannot be displayed here anymore. %1$sRead our statement on this, for further information%2$s.', "wordpress-seo" ),
```

Comment:
A comma after `have` in the second sentence would make the sentence more readable

Reply:
And I think the comma before `for further information` can be removed

Reply:
I don't think the first one would be proper English. The second can definitely be removed.

Reply:
I removed the second comma and left the first alone.